triple-store/triple-store-impl-rdf4j/src/test/java/com/powsybl/triplestore/impl/rdf4j/test/PowsyblWriterSequenceFixTest.java

```diff
@@ -0,0 +1,223 @@
+package com.powsybl.triplestore.impl.rdf4j.test;
+
+import static org.junit.Assert.assertFalse;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+import javax.xml.transform.Source;
+
+import org.eclipse.rdf4j.IsolationLevels;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.Literal;
+import org.eclipse.rdf4j.model.Resource;
+import org.eclipse.rdf4j.model.Statement;
+import org.eclipse.rdf4j.model.vocabulary.RDF;
+import org.eclipse.rdf4j.repository.RepositoryConnection;
+import org.eclipse.rdf4j.rio.RDFHandlerException;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xmlunit.builder.DiffBuilder;
+import org.xmlunit.builder.Input;
+import org.xmlunit.diff.Diff;
+
+import com.powsybl.commons.datasource.DataSource;
+import com.powsybl.commons.datasource.MemDataSource;
+import com.powsybl.triplestore.api.PropertyBag;
+import com.powsybl.triplestore.api.PropertyBags;
+import com.powsybl.triplestore.impl.rdf4j.TripleStoreRDF4J;
+
+public class PowsyblWriterSequenceFixTest {
+
+    PropertyBags objects = new PropertyBags();
+    List<String> objectProperties = Arrays.asList("id", "property1", "property2");
+    String namespace = "http://test/";
+    String contextName = "context1";
+    String qualifiedContextName = "http://test/" + contextName;
+    String objectType = "http://test/type1";
+    String expected = "/fix-powsybl-writer-objects.xml";
```

Comment:
Should all these variables be private and maybe final?

Reply:
agree